internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -284,6 +281,47 @@ func (opts *Opts) askClusterOptions() error {
 	return nil
 }
 
+func (opts *Opts) defaultRegions() ([]string, error) {
+	cloudProviders, err := opts.store.CloudProviderRegions(opts.ConfigProjectID(), opts.Provider, tier, false)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if len(cloudProviders.Results) == 0 || len(cloudProviders.Results[0].InstanceSizes) == 0 {
+		return nil, fmt.Errorf("no regions available")
+	}
+
+	availableRegions := cloudProviders.Results[0].InstanceSizes[0].AvailableRegions
+
+	var defaultRegions []string
```

Comment:
[not blocking] FYI, you can still use make, 
```suggestion
	defaultRegions := make([]string, 0, len(availableRegions))
```

the third param is the capacity of the array but the it still has a len of zero so append works as you may expect
no need to change anything here just an FYI